docs/adr/0004-adr-process.md

```diff
@@ -0,0 +1,32 @@
+# 4. ADR Process
+
+Date: 2018-12-07
+
+## Status
+
+Approved
+
+## Context
+
+We need a documented process for proposing, discussing and ultimate accepting or
+rejecting ADRs via pull requests.
+
+## Decision
+
+We have decided to adopt a process that favours rapid changes, at least while
+the project is in its infancy. To this end, we will allow ADRs in the `proposed`
+status to be merged to `master` via PRs. The PRs may contain related code changes.
+
+Whenever a `proposed` ADR is merged to `master`, a GitHub issue is created to
+capture discussion about the ADR. Such issues are given the `adr` label.
+
+Any `proposed` ADRs remaining on `master` must be resolved either by approving
+the ADR, or by rejecting it and reverting any associated code changes.
+
+## Consequences
+
+This approach favours allows project maintaners to proceed with changes quickly,
+documenting their decisions, and giving us standard way to discuss those decisions.
```

Comment:
This paragraph needs work.

Something is not right with "favours allows". Also "maintaners" is spelt wrong? 
> This approach favours allows project maintaners

Missing "a"?
> and giving us _a_ standard way to